utils/whois/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=whois
+PKG_VERSION:=5.5.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/rfc1036/whois/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=ae389c1486cdeae99f5f02940f98f5d7ff1f08ac0f96810365159b27b0189b5e
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/whois
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Intelligent WHOIS client
+  DEPENDS:=
+  URL:=https://github.com/rfc1036/whois
+endef
+
+define Package/whois/description
+This client is intelligent and can automatically select the appropriate whois
+server for most queries.
+
+The internal database is often more accurate than IANA's published one, but
+please send me any information you have regarding domains and network resources
+which are not correctly handled by the program.
+endef
+
+EXTRA_CFLAGS+=-DHAVE_GETOPT_LONG
+MAKE_FLAGS += PERL=$(STAGING_DIR_HOST)/bin/perl
```

Comment:
This will fail on systems that do not have autodie.pm installed.

I don't know if it's worth adding a build dependency on perl/host.

Reply:
I don't know what that is. If you don't consider it problematic, please merge. If not I'll add perl/host.

Reply:
So you are upstreaming (to OpenWrt) something which you don't know what it is doing or why it should be there? Wow.

Reply:
@BKPepe I'd appreciate if you adjust your attitude. I'm porting `whois` which happens to contain `mkpasswd`.

> Because of historical reasons this package also contains the mkpasswd
program, which can be used to encrypt a password with crypt(3).

I patched the Makefile so it shouldn't be compiled anymore, which I expected you to figure out on your own.

Reply:
@BKPepe I'd appreciate if you could adjust your attitude. CI builds don't fail with the default packages installed. If there are corner cases where `autodie.pm` is missing, please elaborate.

My local Debian 10 did not require an additional package :shrug:

Reply:
I adopted the commit based on @neheb feedback here and via IRC, thanks again!